lib/FieldType/Mapper/DateTimeFormMapper.php

```diff
@@ -9,45 +9,109 @@
-                ],
-                'choices_as_values' => true,
-                'expanded' => true,
-                'required' => true,
-                'property_path' => 'fieldSettings[defaultType]',
-                'label' => 'field_definition.ezdatetime.default_type',
-            ])
-            ->add('dateInterval', DateTimeIntervalType::class, [
-                'required' => false,
-                'property_path' => 'fieldSettings[dateInterval]',
-                'label' => 'field_definition.ezdatetime.date_interval',
-            ]);
+            ->add(
+                'useSeconds',
+                CheckboxType::class,
+                [
+                    'required' => false,
+                    'property_path' => 'fieldSettings[useSeconds]',
+                    'label' => 'field_definition.ezdatetime.use_seconds',
+                ]
+            )
+            ->add(
+                'defaultType',
+                ChoiceType::class,
+                [
+                    'choices' => [
+                        'field_definition.ezdatetime.default_type_empty' => Type::DEFAULT_EMPTY,
+                        'field_definition.ezdatetime.default_type_current' => Type::DEFAULT_CURRENT_DATE,
+                        'field_definition.ezdatetime.default_type_adjusted' => Type::DEFAULT_CURRENT_DATE_ADJUSTED,
+                    ],
+                    'choices_as_values' => true,
+                    'expanded' => true,
+                    'required' => true,
+                    'property_path' => 'fieldSettings[defaultType]',
+                    'label' => 'field_definition.ezdatetime.default_type',
+                ]
+            )
+            ->add(
+                'dateInterval',
+                DateTimeIntervalType::class,
+                [
+                    'required' => false,
+                    'property_path' => 'fieldSettings[dateInterval]',
+                    'label' => 'field_definition.ezdatetime.date_interval',
+                ]
+            );
+    }
+
+    /**
+     * @param FormInterface $fieldForm
+     * @param FieldData $data
+     */
+    public function mapFieldValueForm(FormInterface $fieldForm, FieldData $data)
+    {
+        $fieldDefinition = $data->fieldDefinition;
+        $fieldSettings = $fieldDefinition->getFieldSettings();
+        $formConfig = $fieldForm->getConfig();
+        $names = $fieldDefinition->getNames();
+        $label = $fieldDefinition->getName($formConfig->getOption('languageCode')) ?: reset($names);
+
+        $fieldForm
+            ->add(
+                $formConfig->getFormFactory()->createBuilder()
+                    ->create(
+                        'value',
+                        DateTimeType::class,
+                        [
+                            'input' => 'datetime',
+                            'date_widget' => 'single_text',
+                            'time_widget' => 'single_text',
```

Comment:
will that generate two inputs (`date` + `time`) or one `datetime` input ?
btw, in https://github.com/ezsystems/hybrid-platform-ui/pull/65/files#diff-46b5c70bde48cbbdc0324e8043b3be5cR378, it's specified this should be a `datetime-local` input (that does not seem to be possible out of the box according to https://symfony.com/doc/current/reference/forms/types/datetime.html)